### PR TITLE
LPS-130243 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-130243

<html>
<body>

Test File Name | Refactorneeded | New add or spilt | Removed | Move in | Move out | Total (Refactordone) | Comment
-- | -- | -- | -- | -- | -- | -- | --
DMSharing.testcase | 12 | 3 | 2 |   |   | 13 | Spilte test  DMSharing#AssertNotifications to /DMSharing#CanShareWithUpdatePermissions /DMSharing#CanShareWithCommentPermissions /DMSharing#CanShareWithViewPermissions  Remove GoToSharedContentViaNotifications and it duplicates CanShareWithViewPermissions
SharingUsecase.testcase | 4 |   |   |   |   | 4 |  
AssetsSharingUpgrade.testcase | 2 |   |   |   |   | 2 |  
Total |   |   |   |   |   | 19 |  

</body>
</html>

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Review by austinchiang — COMMENTED** [review]
@Jeremy-Ch  Next time please be more specific with saying what you're refactoring and commit them in smaller chunks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/assetsharing/DMSharing.testcase [hunk 1]
```diff
@@ -56,181 +52,196 @@ definition {
 		}
 	}
 
-	@description = "This test asserts that users get the proper notification based on sharing permissions granted to him, also covers LPS-94448"
-	@priority = "4"
-	@refactorneeded
-	test AssertNotifications {
-		DMDocumentSharing.addDocumentWithOnlyOwnerPermissions(
+	@description = "This ensures that sharing a document to a user, and changing its sharing permissions via Manage Collaborators. This test also covers permissions and making sure users only have access to sharing permissions assigned to him."
```
[the remark this answers is not in the document]
Comment (Jeremy-Ch) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/assetsharing/DMSharing.testcase:55]: I will work on this with [LPS-141188](https://issues.liferay.com/browse/LPS-141188)

**Comment by liuxingye** [comment]
ci:close

**Pull request closed without merging**